uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/components/my-leave-calendar.js

```diff
@@ -93,7 +96,7 @@ define([
      * @return {Promise}
      */
     vm._loadLeaveRequestsAndCalendar = function () {
-      return parentCtrl._loadLeaveRequestsAndCalendar.call(vm, 'contact_id', true);
+      return parentCtrl._loadLeaveRequestsAndCalendar.call(vm, 'contact_id', false);
```

Comment:
isn't there a default value for the second parameter? so that you can omit specifying it?